spaces/S000030/README.md

```diff
@@ -13,17 +14,27 @@ refs:
     name: Hilbert space on Wikipedia
   - doi: 10.1090/S0002-9904-1968-12044-0
     name: A complete elementary proof that Hilbert space is homeomorphic to the countable infinite product of lines (Anderson & Bing)
+  - mr: 0478168
+    name: Selected topics in infinite-dimensional topology (Bessaga & Pełczyński)
 ---
 Let $X$ be the space $\ell^2$ of all sequences $x = \langle x_i \rangle$ of real numbers such that $\Sigma x_i^2$ converges, with topology generated by the metric $d(x,y) = \sqrt{\Sigma(x_i - y_i)^2}$.  Any separable Hilbert space is isomorphic to $\ell^2$.
 
 This space is homeomorphic to $\mathbb{R}^\omega$ with the product topology (Fréchet space).  For this deep result due to R. D. Anderson, see for example {{doi:10.1090/S0002-9904-1968-12044-0}}.
 
-More generally, any infinite-dimensional separable Banach space or separable Fréchet space is homeomorphic to $\mathbb R^\omega$.  See {{wikipedia:Anderson–Kadec theorem}}.
+More generally, any infinite-dimensional separable Banach space or, even more generally, any infinite-dimensional separable Fréchet space is homeomorphic to $\mathbb R^\omega$.  See {{wikipedia:Anderson–Kadec theorem}}.
```

Comment:
Theorem VI.5.2 (Anderson-Kadec) on page 189 of [Bessaga-Pełczyński] states 
> Every infinite-dimensional separable Fréchet space is homeomorphic to $\mathbb{R}^\omega$.

So perhaps the following is an improvement? Feel free to reject this / reword. The Wikipedia page mentions page 189 in a footnote but doesn't give the theorem number, so this would be slightly more reassuring.

> This is stated in Theorem VI.5.2 (Anderson-Kadec) on page 189 of {{mr:0478168}}. Also see {{wikipedia:Anderson–Kadec theorem}}.